scripts/build/Jenkins/Jenkinsfile

```diff
@@ -401,29 +577,27 @@ def PreBuildCommonSteps(Map pipelineConfig, String snapshot, String repositoryNa
     // we need to cleanup. Once all branches take this relocation, we can remove this
     if(env.CLEAN_WORKSPACE?.toBoolean() || fileExists("${workspace}/.git")) {
         if(fileExists(workspace)) {
+            echo 'Clean workspace...'
             PlatformRmDir(workspace)
         }
     }
 
-    def workspace_retry = 5
```

Comment:
@amzn-changml discovered an issue where EBS volume may not be immediately available for writes once mounted which is the reason I think the retry exists. Is this issue resolved?

Reply:
I removed the retry because it was causing my builds to fail.

Reply:
Got it. We can remove it if this is causing issues with the updated folder setup.

Reply:
I curious on what the failure was here, but fine with removing it if the prebuild setup is implicitly looking for the presence of the workspace folder.